nautobot/extras/models/groups.py

```diff
@@ -194,9 +194,20 @@ def _map_filter_fields(self):
                 logger.debug("Skipping %s: doesn't have a filterset field", missing_field)
                 continue
 
+            # Skip filter fields that have methods defined. They are not reversible.
+            if filterset_field.method is not None:
```

Comment:
@jathanism We also need this for the fields sourced from `filterform_fields = filterform.fields` - can You please ensure all of the the logic is same for missing and filterform_fields ?

Reply:
Huh I thought that was covered. Thanks I'll double check.